### DISCUSSION TITLE
Show miss location from replay editor

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#14720] [Ideas] [answered]
actually in lazer we can progress through the replay using the bar at the bottom of the screen. 
But what about showing the miss we have on the replay in this bar ?
inspired by this video
https://youtu.be/m6LUF9GjVbc?t=316

lazer
![image](https://user-images.githubusercontent.com/50368878/132984709-a30843ab-8398-48b2-adeb-51ded617cae7.png)

proposal
![Sans titre](https://user-images.githubusercontent.com/50368878/132985067-76cab2b3-4668-4a4f-ad03-e10179d2bdaf.png)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/5681

**Reply by Biokax** [reply]
Well it seen to have no update since 2020 did it progress a bit, or just waiting for redesign for now ?

**Reply by frenzibyte** [reply]
I don't understand what you're saying, but your proposal is part of the linked issue thread and therefore doesn't have to be tracked.

**Reply by Biokax** [reply]
The linked issue is about progress details in the actual play. The proposal is about showing miss details when you watch a replay to know where you miss.

**Reply by bdach** [reply]
which are the same thing.

linked issue is low priority. completing the game in terms of parity with stable comes first.
